docs/api/topics/firefox_os_addons_review.rst

```diff
@@ -33,27 +35,17 @@ Add-ons Queue
     :status 200: successfully completed.
     :status 403: not authenticated.
 
-
-.. http:get:: /api/v2/extensions/extension/(int:id)|(string:slug)/
```

Comment:
This API is gone for now. Use the regular detail API when you want to access the detail API for an add-on in the queue instead (as reviewer, you should have access)